### ISSUE TITLE
Changed domain of 99liga

### ISSUE THREAD

**Issue by blssed** [flashkillapp/flashkill#81]
**Describe the bug**
They changed the website to liga.esl-meisterschaft.de, so the plugin doesn't work automatically anymore.

**To Reproduce**
1. Go to the league website
2. Open a team page
3. Doesn't work

**Expected behavior**
It should show all the stats

**Comment by blssed** [comment]
I would contribute to this by changing the necessary lines, but I think the domain is hardcoded in a lot of places and I would rather let you change it in an organized manner.

**Comment by timonla** [comment]
If this is just about replacing some URLs, I can take a look at it later today.

**Comment by blssed** [comment]
I think it is, I have no experience with writing browser extensions, but the only think 99damage changed, is the domain, so for me it would be logical to update the URI's

**Comment by timonla** [comment]
@blssed Check out #82. reactivating the extension was, like you suggested, just changing a few URLs. However, I noticed that there are some more changes with the switch to ESL Meisterschaft. Since I am not active in the community right now, I don't see these things right away.  I opened #85 but unfortunately don't have time to work on this right now.

Version 2.2.0 is published and at least activates the extension again.
